followthemoney/schema/Occupancy.yaml

```diff
@@ -0,0 +1,40 @@
+Occupancy:
+  label: "Occupancy"
+  plural: "Occupancies"
+  extends: Interval
+  matchable: false
+  description: >
+    The occupation of a position by a person for a specific period of time.
+  featured:
+    - holder
+    - post
+  required:
+    - holder
+    - post
+  edge:
+    source: holder
+    label: holds
+    target: post
+    directed: true
+    caption: 
+      - startDate
+      - endDate
```

Comment:
does anything bad happen if none of the caption fields have a value? I guess if it did, there ought to be a check that at least one of them is required.

Reply:
The Python and TS libs will fall back to the schema label if there is no caption property with a value.

So if you have an `Occupancy` with start date `2023-07-21` and end date `2023-07-31`, it would display `2023-07-21` as the edge label. If you have an `Occupancy` without a start or end date, it would display just `Occupancy` as the label.